pandas/tseries/offsets.py

```diff
@@ -972,21 +972,25 @@ class BusinessMonthBegin(MonthOffset):
 
 class _CustomBusinessMonth(_CustomMixin, BusinessMixin, MonthOffset):
     """
-    DateOffset subclass representing one custom business month, incrementing
-    between [BEGIN/END] of month dates.
+    DateOffset subclass representing custom business month(s).
+
+    The count increase between %(bound)s of month dates.
```

Comment:
Hmm actually can you retain the original wording here? Something like:

```suggestion
    Increments between %(bound)s of month dates.
```

Reply:
Fixed!